web/css/chat-room.css

```diff
@@ -0,0 +1,311 @@
+
+/** chat room styles*/
+
+
+.chat-room .chat{
+    margin-top: auto;
+    margin-bottom: auto;
+}
+.chat-room .card{
+    height: 500px;
+    border-radius: 15px !important;
```

Comment:
Too many `!important`.

`!important` is rarely needed as there are generally other ways to achieve the required behavior by just applying CSS rules priority. See https://uxengineer.com/css-specificity-avoid-important-css/ for learning about that.

Reply:
Thanks for info, its clear now.